CMakeLists.txt

```
@@ -117,6 +115,37 @@ target_include_directories(musica
 
 add_subdirectory(src)
 
+#
+# jiwon 7/21 - in progress
+#
+####################
+# MUSICA - fortran
+add_library(musica-fortran STATIC src/fortran_interface/micm/interface.F90)
```

```suggestion
add_library(musica-fortran)
```

The file itself will be added with the suggested change to the CMakeLists file in srd/fortran_interface